com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -397,8 +397,8 @@ static IEnumerable<string> CheckOutputTensorPresence(Model model, int memory)
             BrainParameters brainParameters, TensorProxy tensorProxy,
             SensorComponent[] sensorComponents, int observableAttributeTotalSize)
         {
-            // TODO: Update this check after intergrating ActionSpec into BrainParameters
-            var numberActionsBp = brainParameters.VectorActionSize.Length;
+            var numberActionsBp = brainParameters.VectorActionSpec.NumContinuousActions +
```

Comment:
How do we handle previous actions with hybrid actions? This makes it look like we concatenate them into a single array of floats. Do we need to change the trainer to have separate previous continuous and previous discrete?

Reply:
Previous actions is only used with discrete+LSTM. 
Fixed this to discrete sizes only.